src/test/miner_tests.cpp

```diff
@@ -136,6 +138,96 @@ struct {
+    {"00000000000000000000000000000000000000000000000000000000000018c5", "003f7160ba1eb3d1992200815707240d1f0ff76d040a3f7993f664545b9a3735cf4fd9262e3a577f56e713dc6b114e209751db2523f13c6de6b1545c5065d3350a0b8b4050abd2cc77760db670fb7a923b999ab803730ceefb4ed7955953b3efddb07c71cfd0de19bd356b9f3f2b5259f9d79d36a3b2dc50ce1f88d3d5623e34ef76ee546cfd827484a16d5330852bc2b90cef44a3bf1b8c6c309bc259292fa157582e7be6bd687a00a23c9a549ce20b19f1c14d9bc2f31703afdde89318372c9a1d882933350e343078a978cf25433d89c70582e1fc6b2bce256123a3582cda5bf21be7bf0aa806bc47dbd6343697d94432bc6f75981e7e871a4e9c0b726ebcad1675f8e1f285c5e365cb79d30e917a5d26bacbfcaf288c69f68d4475dc4482e9c31dda254410c8de06a00ca3993e9011973c3f900e0b917764a413e30b22a61b4a2941cef7b6d8f53ce25928beec6d022733c532e379c924ff02f63ff66110e92529202d267a53e515cd545aefe6f59cf059bdc346c71ecc9b0e6d6130acb0aea1c70441560eb6731f6a68deaa3441c9a6ec4d6495e19014057d8fba02ce52131cd01a0237aaba41a075d58b18e19e49358ec20efa51d43709606dfa45e0a019f94c6118c8d8fc205ca8638f0f051eac8562e7d2efaf5433103856b91d383fb7723e16f385aa7ccc09932d7643a48038d3cdeec4de7e90035b2f4c4c9574e9e35a506ea39d4b38479915bf6b0365ea0683b018ddb16f01f251e5a475f7593dc6002e65d74013ad6117a33133d4fd3c83850fa2e8cf1836db06c8b5d189c182f0853ba455669308f559ec140572f93a97e24d6f57cbb1b43dfa8bee3700f2f89c0ec079c5411aad0d44a711c2dada6a6d035faa9883118741b3055fdce93d12078e05620fb6487ef2d6f41abe5e7d70f66c79ebfce5471762348a716e3852f20094e75ecdd2a37ae291003b571d9508d0f1fbf57b1bb7871e98d9fe1b889b852503deeee95c3afa06a419fd834431c984506bf454dc31fbb7b19c8ad3be5f2f870df9a7d8cb030b43f38dae543015106a5faef5062cc7db6a1bc065091260ac5b1b85d48935ae29d8072e2beceb129510f97bc46e0d58b605e78e52dad908d91d678e9f2527776971fbb6eedc297aeb8f150573f4efc5a36d942fa1a6c7983e76e7a20e6e3913b200fded49f88fb5a888f9754705df0606699173acec199f93cca550053294a68286e03b3c9dab6c8e3f2620ca17e0980f754183ac621e0a9c71dccd4d38ab0c264ec3d79b22608b2f55c7feea420adae96d786d650a404e9b834874f8ed57d4f8df5aedc6cf041bf853779147b323ababc3f0daf840305741ee306d3fdd031704c13b8e0e6e28d46ec270e5ffe2f7146edfb5ad762e56e520a5fcadacf92cf955739f43be15dfcb170243f0ef7d861b29a9c521e5ef70d1b71946fcca2538036ea92ddb221b95acd4dd9bd55dbdab184e51350ab40e6d6a2203932fb0dbe818f54a4eff13bf08431aee5fb10d5cde7130d7b7cfcb6e4c2eedebde28b607213347bad53676f1317496f2a5f61178fbef07600b693ae7285075ad949b932d6fd57571b3880fd82c25611323022f3c79a29c5ab08e638053603bfc341345a5b656951ceb3b7bccb70fedf47e5e1e1ddd4c5d0286875afcc4cdd31bd1c4e7424036f96e7a9d62ad107743441a6369af57fdd61197d3163f471f5b1c7205d8c6f908d43db9d8d512fed1bd041a2b16dfb98716eb442589074df11641d330585366aaf5b51bed51033c590eceda8e3bfed790572132977482c246708e2d73b28ba01d3707fbf27949bc5e9baed5a41ae80406eed93198cae9c548b9316768ba7a91652551080321167c92a996d6e35030938410e42f6a79bdd44512"},
```

Comment:
I have not checked any of these, and I'm not sure how I would. A comment identifying the source of these serialized transactions would be helpful.

Reply:
Will do. They were CPU-mined by the code in this method.

Reply:
Note that they are not transactions, they are miner nonces and Equihash solutions. `CreateNewBlock_validity` is recreating blocks from them via `ProcessNewBlock`, and that would fail if they were incorrect.